Automattic/newspack-blocks
# fix(modal-checkout): prevent initial render of details table

### All Submissions:






### Changes proposed in this Pull Request:



Prevents initial render of the order details table.  

See 1200133283036252-as-1205933702651194

### How to test the changes in this Pull Request:

1. On `master`, make a donation using the modal checkout – observe that the order details table is visible for a moment right after the modal content loads
2. Switch to this branch, observe the issue solved 
3. [Enable coupons](https://woo.com/document/coupon-management/) and create a coupon. Apply it during the checkout and observe the order details table appears only after the coupon is applied

### Other information:
